docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: '3.8'
+
+services:
+  postgres:
+    image: postgres:16-alpine
+    container_name: toolhive-registry-postgres
+    environment:
+      POSTGRES_USER: registry
+      POSTGRES_PASSWORD: registry_password
+      POSTGRES_DB: registry
+      POSTGRES_INITDB_ARGS: "-E UTF8"
+    ports:
+      - "5432:5432"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+      - ./database/migrations/000001_init.up.sql:/docker-entrypoint-initdb.d/01-schema.sql:ro
```

Comment:
this is fine for loading the initial schema, but where would the migrations run? wouldn't you need an explicit `migrate` command and an explicit container in between postgres and registry-api?